client/routes/workflow/index.vue

```diff
@@ -298,7 +298,7 @@ export default {
         });
     },
     onQueryUrlChange(queryUrl) {
-      this.fetchHistoryPage(queryUrl);
+      setTimeout(() => this.fetchHistoryPage(queryUrl), 100);
```

Comment:
this change was added for the integration tests were locking up and not letting the javascript thread request the feature flags.

Reply:
Do you need 100ms delay, or will 0ms work?

Can the tests be mocked in a different way?

Reply:
I can reduce delay to 30ms and it still passes. In practice this delay won't be noticable as this is for when fetching the next page of data. if anything we may want to increase this delay because at the moment it keeps fetching as soon as the previous response has been fetched which can be abit aggressive on browser CPU usage.

Reply:
Looks like I can work around this by adding a delay to the mocked api. I need to update the fetch-mock library to include delay.

Reply:
Removed timeout and added delay to mockRequest in integration test.